datafusion/optimizer/src/optimizer.rs

```diff
@@ -178,16 +178,15 @@ impl Optimizer {
         F: FnMut(&LogicalPlan, &dyn OptimizerRule),
     {
         let mut new_plan = plan.clone();
-        debug!("Input logical plan:\n{}\n", plan.display_indent());
-        trace!("Full input logical plan:\n{:?}", plan);
+        log_plan("Optimizer input", plan);
```

Comment:
This is a driveby cleanup to improve logging (specifically, also add `trace!` to log schema)